ffi/tests/ctests/src/tests.c

```diff
@@ -0,0 +1,221 @@
+#ifdef NDEBUG
+#undef NDEBUG
+#endif
+
+#include <stdlib.h>
+#include <stdbool.h>
+#include <string.h>
+#include <assert.h>
+
+#include <wirefilter.h>
+
+#define WIREFILTER_STRING(v, s) {(v).data = (const unsigned char *)(s); (v).length = strlen(s);}
```

Comment:
Why did you keep both the macro and the function? One should be enough.

Reply:
It's just a leftover I forgot to remove :-)